setup.py

```diff
@@ -22,6 +22,9 @@
         exclude=["*.tests", "*.tests.*", "tests.*", "tests"]
     ),
     include_package_data=True,
+    install_requires=[
+        "django-rich>=1.0,<2.0",
```

Comment:
Have you read... [“Should You Use Upper Bound Version Constraints?”](https://iscinumpy.dev/post/bound-version-constraints/) ? 😄

Reply:
I have not. Do you wanna PR this?